utilities/decrypt_data/export_postgres_to_excel_chunks.py

```
@@ -0,0 +1,585 @@
+            
+            # Generate filename based on schema
+            if schema == 'ifix':
+                excel_filename = os.path.join(output_dir, f"ifix_{table_name}.xlsx")
+            else:
+                excel_filename = os.path.join(output_dir, f"{table_name}.xlsx")
+            
+            result['filename'] = excel_filename
+            
+            # Calculate number of chunks needed
+            total_chunks = (db_row_count + chunk_size - 1) // chunk_size
+            logger.info(f"Table {schema}.{table_name}: Processing {db_row_count:,} rows in {total_chunks} chunks of {chunk_size:,} rows each")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)